storagetransfer/storagetransfer_test.go

```diff
@@ -151,12 +151,14 @@ func TestQuickstart(t *testing.T) {
 	tc := testutil.SystemTest(t)
 
 	buf := new(bytes.Buffer)
```

Comment:
I think the buffer alloc and check should also go inside the retry loops (here and elsewhere). This means that we retry if the output is unexpected as well.